roles/apt-repos/tasks/main.yml

```diff
@@ -3,13 +3,13 @@
   apt_key:
     url: "{{ item.key_url }}"
     validate_certs: "{{ item.validate_certs|default(omit) }}"
-  with_items: repos
+  with_items: "{{ repos|default([]) }}"
```

Comment:
If we have this defined in `apt-repos/defaults/main.yml` do we need it added on every usage? Or are you trying to standardize syntax?